### PR TITLE
Mock v3

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by praiskup** [comment]
The only test failures are for the Mageia chroots:
```
...
PASSED: mageia-9-i586.cfg (tmpfs)
PASSED: mageia-9-i586.cfg
FAILED: mageia-9-x86_64.cfg (tmpfs)
FAILED: mageia-9-x86_64.cfg
PASSED: mageia-cauldron-i586.cfg (tmpfs)
PASSED: mageia-cauldron-i586.cfg
FAILED: mageia-cauldron-x86_64.cfg (tmpfs)
FAILED: mageia-cauldron-x86_64.cfg
PASSED: navy-8-x86_64.cfg (tmpfs)
...
```
Example:
```
Mageia Cauldron - x86_64                        137  B/s | 682  B     00:04
Errors during downloading metadata for repository 'mageia-cauldron':
  - Curl error (78): Remote file not found for ftp://ftp-stud.hs-esslingen.de/pub/Mirrors/Mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz [The file does not exist]
  - Curl error (78): Remote file not found for ftp://mageia.jameswhitby.net/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz [The file does not exist]
  - Status code: 404 for http://mirror.math.princeton.edu/pub/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 128.112.18.21)
  - Curl error (78): Remote file not found for ftp://mageia.jameswhitby.net/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz [The file does not exist]
  - Status code: 404 for https://distro.ibiblio.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 152.19.134.43)
  - Status code: 404 for http://distro.ibiblio.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 152.19.134.43)
  - Status code: 404 for http://mageia.jameswhitby.net/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 72.82.55.97)
  - Status code: 404 for https://mirrors.kernel.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 198.145.21.9)
  - Status code: 404 for http://mirrors.kernel.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 198.145.21.9)
  - Status code: 404 for http://mirror.math.princeton.edu/pub/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 128.112.18.21)
  - Status code: 404 for http://mageia.jameswhitby.net/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 72.82.55.97)
  - Status code: 404 for http://mirrors.kernel.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 198.145.21.9)
  - Status code: 404 for https://mirror.math.princeton.edu/pub/mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 128.112.18.21)
  - Curl error (78): Remote file not found for ftp://ftp-stud.hs-esslingen.de/pub/Mirrors/Mageia/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz [The file does not exist]
  - Status code: 404 for https://mirrors.kernel.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 198.145.21.9)
  - Status code: 404 for https://mageia.jameswhitby.net/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 72.82.55.97)
  - Status code: 404 for https://distro.ibiblio.org/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 152.19.134.43)
  - Status code: 404 for https://mirror.math.princeton.edu/pub/mageia/distrib/cauldron/x86_64/media/core/release/repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz (IP: 128.112.18.21)
  - Downloading successful, but checksum doesn't match. Calculated: 664f2b1654c363a6348b688d5d475ed9ec0e7ef3c72f6f315f37fe97a2fe63eb(sha256)  Expected: dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8(sha256)
  - Status code: 404 for https://mageia.jameswhitby.net/distrib/cauldron/x86_64/media/core/release/repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz (IP: 72.82.55.97)
Error: Failed to download metadata for repo 'mageia-cauldron': Yum repo downloading error: Downloading error(s): repodata/dd57264d9881b08d55c74d437d06e31166a7d4392d91bfafa65078c78a686cd8-primary.xml.gz - Cannot download, all mirrors were already tried without success; repodata/8fba04cc6aa852fbc18bf625458885033b766f7762ca4ce2550b14b2e0c9e70a-filelists.xml.gz - Cannot download, all mirrors were already tried without success
```

This is not a blocker for the release, doesn't seem to be problem in mock.

File: .tito/packages/mock [hunk 1]
```diff
@@ -1 +1 @@
-2.16-1 mock/
+3-1 mock/
```
Comment (Conan-Kudo) [inline .tito/packages/mock:1]: This should be `3.0`, not `3`

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

File: .tito/packages/mock (hunk 1 above)
Comment (praiskup) [reply .tito/packages/mock:1]: Why?

Comment (praiskup) [reply .tito/packages/mock:1]: I see:
```
rpmdev-vercmp 3-1 3.0-0
3-1 < 3.0-0
```
And we have added `Requires: mock >= 3.0` in fedora-review :-(

**Pull request closed without merging**